src/lib.rs

```diff
@@ -104,10 +104,7 @@ pub fn private_key(rd: &mut dyn io::BufRead) -> Result<Option<PrivateKeyDer<'sta
             Item::Pkcs1Key(key) => return Ok(Some(key.into())),
             Item::Pkcs8Key(key) => return Ok(Some(key.into())),
             Item::Sec1Key(key) => return Ok(Some(key.into())),
-            Item::X509Certificate(_)
-            | Item::SubjectPublicKeyInfo(_)
-            | Item::Crl(_)
-            | Item::Csr(_) => continue,
+            _ => continue,
```

Comment:
Perhaps extract these fixes in a separate commit, since this seems like a logically independent change?

Reply:
(fell away, it was due to the `Item` types `non_exhaustive`ness being activated by it being in a different crate)